app/scripts/satisfaction.js

```diff
@@ -1,11 +1,20 @@
-window.onload = function () {
+jQuery(document).ready(function($) {
```

Comment:
Why are you referencing jQuery globally, instead of importing it since you are using a module system?